sqlite3/example/multiplatform/db/sqlite3/web.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:sqlite3/common.dart' show CommonDatabase;
+import 'package:sqlite3/wasm.dart' show IndexedDbFileSystem, WasmSqlite3;
+
+Future<CommonDatabase> openSqliteDb() async {
+  const name = 'my_app';
+  final sqlite = await WasmSqlite3.loadFromUrl(Uri.parse('sqlite3.wasm'));
```

Comment:
I think it may be helpful to mention that this requires an additional file to be added to the project, for instance with a comment pointing back to the readme.

Reply:
Thanks, done: https://github.com/simolus3/sqlite3.dart/pull/189/commits/1d436749fd7915c9cf2254bb2a1309ed4c513890